redesign/careers.html

```diff
@@ -10,7 +10,7 @@
     <!-- End Google Tag Manager -->
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="dwyl is a worldwide community of people using technology to solve real problems.">
+    <meta name="description" content="Meet our core team of developers, designers, scrum masters and more.">
```

Comment:
this is the careers page, so perhaps:

Learn what it's like to be a developer, designer or scrum master at our open source product development agency